include/openssl/x509.h

```diff
@@ -517,6 +518,9 @@ EC_KEY *d2i_EC_PUBKEY(EC_KEY **a, const unsigned char **pp, long length);
 DECLARE_ASN1_FUNCTIONS(X509_SIG)
 void X509_SIG_get0(X509_ALGOR **palg, ASN1_OCTET_STRING **pdigest,
                    X509_SIG *sig);
+void X509_SIG_get0_const(const X509_ALGOR **palg,
```

Comment:
It's also a little more common to call them `get0_some_thing` though I don't feel too strongly about that, esp when retrieving multiple components.